docs/src/code/docs-style-guide.adoc

```diff
@@ -1,185 +1,256 @@
+// FIXME Do we need/want a more formal workflow ?
+//the applicable https://about.gitlab.com/handbook/product/categories/#devops-stages[DevOps stage or group].
 Otherwise, forge ahead with your best effort. It does not need to be perfect;
 the team is happy to review and improve upon your content. Review the
-xref:index.adoc[Documentation guidelines] before you begin your first documentation MR.
+// TODO Development development front page
+xref:index.adoc[Documentation guidelines] before you begin your first
+documentation PR.
 
-Maintaining a knowledge base separate from the documentation would
-be against the documentation-first methodology, because the content would overlap with
+Maintaining a knowledge base separate from the documentation would be against
+the documentation-first methodology, because the content would overlap with
 the documentation.
+////
 
-== Markdown
-
-All GitLab documentation is written using https://en.wikipedia.org/wiki/Markdown[Markdown].
+[[docs:style:asciidoc]]
+== AsciiDoc
 
-The https://docs.gitlab.com[documentation website] uses https://gitlab.com/gitlab-org/gitlab_kramdown[GitLab Kramdown],
-a "flavored" Kramdown engine to render pages from Markdown to HTML. The use of Kramdown's
-features is limited by our linters, so, use regular Markdown and follow the rules in the
-linked style guide. You can't use Kramdown-specific markup (for example, `{:.class}`).
+All LinuxCNC documentation is written using
+https://en.wikipedia.org/wiki/AsciiDoc[AsciiDoc].
 
-=== HTML in Markdown
+=== HTML in AsciiDoc
 
-Hard-coded HTML is valid, although it's discouraged from being used. HTML is permitted if:
+Hard-coded HTML is valid, although it's discouraged from being used. +
+HTML is permitted if:
 
-* There's no equivalent markup in Markdown.
+* There's no equivalent markup in AsciiDoc.
 * Advanced tables are necessary.
 * Special styling is required.
 * Reviewed and approved by a technical writer.
 
-=== Headings in Markdown
+=== Headings in AsciiDoc
 
-Each documentation page begins with a level 1 heading (`#`). This becomes the `h1` element when
-the page is rendered to HTML. There can be only *one* level 1 heading per page.
+Each documentation page begins with a level 1 heading (`=`). +
+This becomes the `h1` element when the page is rendered to HTML. +
+There can be only *one* level 1 heading per page.
 
-* For each subsection, increment the heading level. In other words, increment the number of `#` characters
-in front of the heading.
-* Avoid headings greater than `H5` (`#####`). If you need more than five heading levels, move the topics to a new page instead.
-Headings greater than `H5` do not display in the right sidebar navigation.
-* Do not skip a level. For example: `##` > `####`.
+* For each subsection, increment the heading level.
+  In other words, increment the number of `=` characters in front of the
+  heading.
+* Avoid headings greater than `H5` (`=====`). If you need more than five
```

Comment:
I _think_ that we have problems with heading levels > 3. 
See, for example, the many warnings here abour "sect5"
http://buildbot.linuxcnc.org/buildbot/builders/2000.docs/builds/8227/steps/make%20docs/logs/warnings%20%283263%29
And bear in mind that sect1 is the overall document, and therefore === in a signle file is sect4